agent/tool-scripts/base-tool

```diff
@@ -575,7 +575,7 @@ function safe_kill() {
 rc=0
 case "${mode}" in
 install)
-	function require_rpm {
+	function check_required_rpm {
 		_tpn="${1}"
```

Comment:
Um...why is this function definition inside a `case` branch?  (Thanks for renaming it, btw.  ;-) )

Reply:
OK, nevermind:  it's defined there because all of the references to it are in that same branch (the _nested_ `case` confused me...still, it's a bit weird to have a function definition inside a nested structure, no?).

Reply:
Because it is only needed for `--install`?